### PR TITLE
ENG-5076 tenantcode in plugincode generation

### PR DESCRIPTION
**Pull request by antromeo**

### CODE REVIEW

File: src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java [hunk 1]
```diff
@@ -52,7 +53,9 @@
 public class PluginProcessor extends BaseComponentProcessor<PluginDescriptor> implements
         EntandoK8SServiceReportableProcessor {
 
-    public static final String PLUGIN_DEPLOYMENT_PREFIX = "pn-";
+    private static final String PRIMARY_TENANT_CODE = "primary";
```
Comment (firegloves) [inline src/main/java/org/entando/kubernetes/model/bundle/processor/PluginProcessor.java]: Please use the constant `EntandoMultiTenancy.PRIMARY_TENANT` from the new version of entando-k8s-custom-model

File: src/main/java/org/entando/kubernetes/model/bundle/descriptor/plugin/PluginDescriptor.java [hunk 2]
```diff
@@ -108,7 +109,7 @@ public PluginDescriptor(String descriptorVersion,
             DockerImage dockerImage, PluginDescriptorV1Spec spec, String name, String deploymentBaseName,
             String image, String healthCheckPath, String dbms, List<String> roles,
             List<PluginPermission> permissions, String ingressPath, String securityLevel,
-            List<EnvironmentVariable> environmentVariables) {
+            List<EnvironmentVariable> environmentVariables, String tenantCode) {
```
Comment (firegloves) [inline src/main/java/org/entando/kubernetes/model/bundle/descriptor/plugin/PluginDescriptor.java]: Is the tenantCode param required?

**Review by firegloves — APPROVED** [review]
LGTM

File: src/main/java/org/entando/kubernetes/model/bundle/descriptor/plugin/PluginDescriptor.java (hunk 2 above)
Comment (antromeo) [reply src/main/java/org/entando/kubernetes/model/bundle/descriptor/plugin/PluginDescriptor.java]: removed

**Pull request merged**